clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/provider/view/KubernetesJobProvider.groovy

```diff
@@ -34,11 +34,11 @@ class KubernetesJobProvider implements JobProvider<KubernetesJobStatus> {
 
   @Override
   KubernetesJobStatus collectJob(String account, String location, String id) {
-    def credentials = accountCredentialsProvider.getCredentials(account).credentials
-    if (!credentials instanceof KubernetesNamedAccountCredentials) {
+    def credentials = accountCredentialsProvider.getCredentials(account)
+    if (!(credentials instanceof KubernetesNamedAccountCredentials)) {
       return null
     }
-    def trueCredentials = credentials as KubernetesNamedAccountCredentials
+    def trueCredentials = (credentials as KubernetesNamedAccountCredentials).credentials
     def status = new KubernetesJobStatus(trueCredentials.apiAdaptor.getPod(location, id), account)
```

Comment:
You could reach this line with a set of `KubernetesV2Credentials`, and this would fail unexpectedly (since that class has no `apiAdaptor`). I think the proper check is 
```
if (!(credentials?.credentials instanceof KubernetesV1Credentials)) { 
  return null
}
```